frontend/components/TeamSection/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Heading, GradientText, HEADING_OPTIONS } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const TeamGroupSection = ({ data }) => {
+  const { title, cardsData } = data;
+
+  const membersData = cardsData.map(cardData => {
+    return (
+      <div className="w-1/2 mx-auto border border-c1000">
```

Comment:
this is c100 not c1000 for black border but you need gray border

Reply:
it's have opacity so c100 isn't the right color, I added a custom color code 'c1100' with a different opacity